requirements.txt

```diff
@@ -1,9 +1,9 @@
-flask
-requests
-gunicorn
-datetime
-altair
-altair_viewer
-ipython
-pytest
-flake8
+Flask==1.1.2
+requests==2.25.0
+gunicorn==20.0.4
+DateTime==4.3
```

Comment:
`datetime` is part of the standard library so I don't think we need to specify it.

Reply:
> @turbatuandrei I don't seem to get an error? Was the error presenting on Travis CI?

You are right, I removed it again and I'm not getting any errors in heroku logs